# This message is for storing image meta
#  Raymond Kirk (Tunstill) Copyright (c) 2021
#  Email: ray.tunstill@gmail.com

Header header

uint32 height
uint32 width

string encoding
uint8 is_bigendian
uint32 step

string algorithm